cardano-crypto-class/src/Cardano/Crypto/Hash/Class.hs

```diff
@@ -44,6 +44,7 @@ module Cardano.Crypto.Hash.Class
   )
 where
 
+import Control.Monad
```

Comment:
You should be specific about what you are importing.

Reply:
Fixed!